src/App.php

```diff
@@ -105,7 +104,6 @@ public function run($request = null, $response = null)
 
         $event = new HttpFlowEvent('bootstrap', $request, $response);
 
-        $container = $this->getContainer();
```

Comment:
`getContainer()` is only usable if called from outside as only return the `container`, use `$this->container` directly instead.